lib/bootstrap/2.disk.coffee

```diff
@@ -1,23 +1,21 @@
 
-module.exports = header: "Disk", handler: ({options}) ->
+module.exports = handler: ({config}) ->
   # Format the disk into partitions
-  options.format ?= true
+  config.format ?= true
   # Ensure the disk is erased from any previous data
-  options.wipe ?= false
-  return unless options.format
-  @system.execute
-    if: options.wipe
-    cmd: """
-    cryptsetup open --type plain #{options.disk} container --key-file /dev/random
+  config.wipe ?= false
```

Comment:
nikita 1 has json schema which beside enforcing the user configuration can provide default values.